public/js/app/components/SearchFilters.tsx

```diff
@@ -49,17 +50,25 @@ const SearchFilters: React.FC<ISearchFiltersProps> = ({ ancestorItem }) => {
 
   const desktopId = 'local-navigation-title'
 
+  const navigate = useNavigate()
+
   function handleParamChange(params: Params) {
     console.log('params: ', params)
     setState(params)
   }
 
+  function handleBackButton() {
+    navigate({
+      pathname: ancestorItem.href
+    })
+  }
+
   // id="mainMenu" is not a correct id as the search filters aren't really a menu.
   // It puts everything in the correct position for now though.
   return (
     <>
       <div id="mainMenu" className="kth-local-navigation col">
-        <a href={ancestorItem.href} className="kth-button back">
+        <a onClick={handleBackButton} className="kth-button back">
```

Comment:
```suggestion
        <Link to={ancestorItem.href} className="kth-button back">{ancestorItem.label}</Link>
```
An `<a>` tag needs a href prop to be valid (for accessibility reasons etc). I guess you switched to onclick to prevent page from reloading from server? 

One solution for this would be to keep the `href` and the add `onClick` with preventDefault as an enhancement, but the good thing is that react router already has a [Link](https://reactrouter.com/en/main/components/link) component for that. So just replace `<a href= ...>` with `<Link to=...>` instead and an a tag with both href and onclick with nagivate will be insterted.

Reply:
Very good point, I will use Link from react-router-dom for sure 🙌